oliviaruizknott/timer-rails
# Delete items in meeting setup

[Fulfills this card.](https://www.notion.so/Timekeeper-Spec-c4c88e683193435393664464cdef6842?p=72ede7401a4a468c9a6e0bad1dd2a6d8)

Now you can delete an item during meeting setup!

## To Test
1. Start setting up a meeting (https://salvadortimer.herokuapp.com/).
2. On the second screen—where you add agenda items—add a few agenda items.
3. You should see an "x" on the right hand side of the agenda items once you’ve entered at least 2 items. (You should not see the "x" if there is only one item.)
4. Clicking the "x" should remove the item.
